internal/worker/sshserver/types.go

```diff
@@ -0,0 +1,70 @@
+// Copyright 2025 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
```

Comment:
Give the proper copyright here please. I forget the syntax for the comment but I often see it in other packages.